app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdEffect.kt

```diff
@@ -28,4 +28,4 @@ data class OpenPatientSearch(
     val patientPrefillInfo: PatientPrefillInfo?
 ) : ScanSimpleIdEffect()
 
-data class OnlinePatientLookupWithIdentifier(val identifier: Identifier): ScanSimpleIdEffect()
+data class OnlinePatientLookupWithIdentifier(val identifier: Identifier, val patientPrefillInfo: PatientPrefillInfo?): ScanSimpleIdEffect()
```

Comment:
At this point, the model already contains the updated `PatientPrefillInfo`. So, instead of passing it to an effect and then to an event and then finally to the `OpenPatientSearch` effect. Can't we just use the `model.patientPrefillInfo` when dispatching the `OpenPatientSearch` effect?